### DISCUSSION TITLE
Feature Request: chunking streaming input

### DISCUSSION THREAD

**Discussion by mrayva** [duckdb/duckdb#9031] [Ideas]
Exactly what some others requested for clickhouse - https://github.com/ClickHouse/ClickHouse/issues/34846

**Comment by Tishj** [comment]
I honestly think we already do this?
<https://github.com/duckdb/duckdb/blob/main/src/common/pipe_file_system.cpp>

**Reply by mrayva** [reply]
Does not appear to be working though...I am using 
<json-outputting-command>  | ./duckdb -c "SELECT * FROM read_json_auto('/dev/stdin',format='newline_delimited')"

Feels like it is waiting for EOF/pipe closing somewhere ...

**Comment by l1t1** [comment]
http://duckdb.org/docs/archive/0.9.1/api/cli#reading-from-stdin-and-writing-to-stdout

**Reply by l1t1** [reply]
I wonder how to do `cat test.csv | ./duckdb :memory: "SELECT * FROM read_csv_auto('/dev/stdin')"` in windows cmd?
I tried and always got `Error: IO Error: No files found that match the pattern "/dev/stdin"`

**Reply by mrayva** [reply]
I'm not longer testing this feature on Windows. Linux only.
I've given up on getting continuous input ingestion to work. Now, I'm tryin to split streaming input
Ex)   <json producing cmd> | split -l 1000 --filter='~/duckdb/duckdb -c "COPY <table_name>  FROM read_json_auto('\''/dev/stdin'\'')" db.duckdb

..but  COPY does not appear to be working:

_Error: Parser Error: syntax error at or near "read_json_auto"
LINE 1: COPY FS FROM read_json_auto('/dev/stdin')_

**Comment by mrayva** [comment]
Probably I don't  use it right but it still does not work:
some-binary-producing-JSONLN-stream | duckdb -c "INSERT INTO MYTABLE BY NAME SELECT * FROM read_json_auto('/dev/stdin');" myduckdb.duckdb

I expect duckdb to periodically 'flush' / commit to DB so that when I kill entire pipeline and open myduck.duckdb and I can see records in MYTABLE ...But the table is empty....
